python/foxglove-sdk/python/examples/ws-demo.py

```diff
@@ -29,15 +30,15 @@ def main() -> None:
 
     server = foxglove.start_server(port=8765)
 
+    # Log messages having well-known Foxglove schemas using the appropriate channel type.
+    box_chan = SceneUpdateChannel("/boxes")
+    tf_chan = FrameTransformChannel("/tf")
+
+    # Log JSON messages using the JsonEncoder.
+    # A ProtobufEncoder is also available.
```

Comment:
your PR comment said that ProtobufEncoder is not available - did I misunderstand that?

bringing this back to our use cases: for logging arbitrary structs I don't think it matters to the user what the wire encoding is, we should just pick something under the hood.

for users who have some existing pubsub system, they should have existing serialization logic so can pass us preserialized messages and schemas directly.

Reply:
👍 I will update.